zerver/views/invite.py

```diff
@@ -19,10 +20,14 @@
 @has_request_variables
 def invite_users_backend(request, user_profile,
                          invitee_emails_raw=REQ("invitee_emails"),
+                         invited_as_admin=REQ(converter=flexible_boolean, default=False),
```

Comment:
do we need `flexible_boolean`?  I'd prefer to just use `validator=check_bool`, which is our common way of doing similar things...